src/main/kotlin/racingcar/view/ResultView.kt

```diff
@@ -0,0 +1,19 @@
+package racingcar.view
+
+object ResultView {
+    private const val PROGRESS_STRING = "-"
+
+    fun showRacingStage(carProgress: List<Int>) {
+        carProgress.forEach {
+            showProgress(it)
+        }
+        println()
+    }
+
+    private fun showProgress(progress: Int) {
+        repeat(progress) {
+            print(PROGRESS_STRING)
+        }
+        println()
```

Comment:
CharSequence.repeat를 활용하면 아래와 같이 변경이 가능합니다!

```suggestion
        println(PROGRESS_STRING.repeat(progress))
```